src/libraries/Common/src/Interop/Windows/IpHlpApi/Interop.ICMP.cs

```diff
@@ -54,13 +54,21 @@ internal struct IcmpEchoReply
         }
 
         [StructLayout(LayoutKind.Sequential, Pack = 1)]
-        internal struct Ipv6Address
+        internal unsafe struct Ipv6Address
         {
-            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 6)]
-            internal byte[] Goo;
-            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 16)]
+            internal fixed byte Goo[6];
```

Comment:
This can't be the name of the corresponding field(s) in Windows, can it? :-)

Reply:
Fixed